### PR TITLE
Recreated PR

### PR DESCRIPTION
**Pull request by github-tommy**

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+</head>
+<body>
+    <div class="header">
+        <h1>Seattle's premiere dog adoption agency</h1>  
+
```
Comment (jimmynono) [inline blog.html]: You'll want to delete these unneeded extra lines.

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <div class="item1">
+        <nav class="site-nav">
+            <a href="index.html">Home</a>
+            <a href="dog.html">/ Dog</a>
+            <a href="blog.html">/ Blog</a>
+        </nav>
+    </div>
+    </div>
+    <div class="container">
+        <h2>Adopt-A-Dog Blog</h2>
```
Comment (jimmynono) [inline blog.html]: This h2 tag is a little out of place. It looks like it's part of a blog entry

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+    </div>
+
+        <div class="footer">
+            <img src="adoptadog-logo.svg" alt="logo" style="width: 30%">
+            <footer>Nondiscrimatory Policy: Adopt-A-Dog avoids any breed, gender or origin of any dogs at anytime. We do not discriminate in administration of our adoption policies at any time.</footer>
+        </div>
+<hr>
+<small>Copyright 2019 @tommybook under supervision and in collaboration with his mentor, J.Norton-UW.</small>
```
Comment (jimmynono) [inline blog.html]: ha, nice. but not needed :)

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    </div>
+    <div class="container">
+        <img src="images/wyatt-ryan-367017-unsplash.jpg" alt=“blog3” style="float:left; width:30%">
+        <h3>Teach Your Dogs to Fetch!</h3>
+        <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.</p>
+        <p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+    </div>
+
+        <div class="footer">
```
Comment (jimmynono) [inline blog.html]: you're missing the links

File: dog.html [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+</head>
+<body>
+    <div class="header">
+        <h1>Seattle's premiere dog adoption agency</h1>  
+
+    </div>
+    <div class="grid-container">
```
Comment (jimmynono) [inline dog.html]: You don't really need to wrap the nav in a grid container

File: dog.html [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+    <div class="item1">
+        <nav class="site-nav">
+            <a href="index.html">Home</a>
+            <a href="dog.html">/ Dog</a>
+            <a href="blog.html">/ Blog</a>
+        </nav>
+    </div>
```
Comment (jimmynono) [inline dog.html]: indenting is off

File: dog.html [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+            </div>
+        </div> 
+        <div class="col-md-3">
+            <div class="thumbnail">
+                <img src="images/duffy-brook-350225-unsplash.jpg" alt="duffy" style="width:40%">
+                <div class="caption">
+                <p>Duffy</p>
+                <p>Cost to Adopt: $123.45</p>
+                <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                <div class="thumbnail"><a href="images/duffy.jpg">Adopt</a></div>
```
Comment (jimmynono) [inline dog.html]: you'll want to next your html inside of the div on a new line. For example
```
<div>
  <a href="#">Hi</a>
</div>
```

Also, these aren't really links, but buttons. I'd also work to make your alt tags a little more descriptive.

File: blog.html (hunk 1 above)
Comment (github-tommy) [reply blog.html]: Noted. Deleted empty line 15.

File: blog.html (hunk 2 above)
Comment (github-tommy) [reply blog.html]: Corrected and moved h2 tag before div tag. Thank you.

File: blog.html (hunk 4 above)
Comment (github-tommy) [reply blog.html]: Sorry. Not quite sure the missing links. pls explain. thanks. FYI, deleted unnecessary empty line 45..

**Pull request closed without merging**